### DISCUSSION TITLE
Documentation about "tag rules"?

### DISCUSSION THREAD

**Discussion by liar666** [tughi/aggregator-android#96] [Q&A]
Hi,
I can't find any documentation about "tag rules". The only mention I found on the whole web was on this issue:
https://github.com/tughi/aggregator-android/discussions/84
I tried to use "condition"s like: feed-title contains "51" but the "Save" button remains disabled, so the syntax is not accepted.

Where can we find the correct syntax for "tag rules"?

**Comment by Frostbyte8** [comment]
The only place the author documented it was in an old update message found here: https://tughi.github.io/aggregator-android/2020/04/07/preview-027.html

As far as I can tell though, text filtering can't be made to be case insensitive.
